Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs

```diff
@@ -14,16 +14,120 @@ static void Main(string[] args)
             Console.ReadKey(true);
         }
 
-        private static void Test(string args, int expected)
-        {
+        private static void Test(string args, int expected)
+        {
             var result = FindDigit(args).Equals(expected) ? "PASS" : "FAIL";
-            Console.WriteLine($"{args} : {result}");
+            Console.WriteLine($"{args} : {result}");
         }
+        public static int FindDigit(string equation)
+        {
+            int index = equation.IndexOf('*');
+            String a = equation.Substring(0,index);
+
+            int index1 = equation.IndexOf('=');
+            String b = equation.Substring(index+1,index1-index-1);
+
+            String c = equation.Substring(index1+1);
+
+            if(a.Contains('?'))  {
+                float num2 = float.Parse(b);
+                float res = float.Parse(c);
+
+                float num1 = res/num2;
+
+                String result = num1.ToString();
+
+                char[] a1 = a.ToCharArray();
+                char[] res1 = result.ToCharArray();
+
+                if(a1.Length == res1.Length)  {

```

Comment:
This is not really programming. I see so much repetition of the code, even if the function is behaving in the exact same way. Refactor this to have functions instead of procedural straight code.